### ISSUE TITLE
State of project

### ISSUE THREAD

**Issue by hExPY** [hyperion-project/hyperion.ng#515]
Hey guys,

it seems like this project died at 7th december 2017. Am i right? I wasn't even able to find any updates in the forum.
Maybe there are any further developing forks or something else.

Kind regards

**Comment by Paulchen-Panther** [comment]
no known fork that is really continued. Brindosch was the only one who had done something further. but that was a long time ago. Since then, the rest of the pull requests are in the official repository. what's going on with brindosch, no idea

**Comment by Paulchen-Panther** [comment]
![screenshot_20180829_221223](https://user-images.githubusercontent.com/16664240/44812862-a1c30b80-abd8-11e8-8b11-d6f85a92fe76.png)

**Comment by pattyland** [comment]
Seems like @brindosch [left the project](https://github.com/orgs/hyperion-project/people)?

At least there is still an admin who can accept pull request ;)

**Comment by pattyland** [comment]
@RickDB @brindosch can't add you: https://github.com/orgs/hyperion-project/people

I guess you need to contace @WoLpH

**Comment by WoLpH** [comment]
It seems I'm the only one that's publicly part of the project but I'm not an admin.

We need to ask one of the following: @brindosch @tvdzwan @poljvd 

The Hyperion-Bot has admin access too but it's harder to get him to reply ;)

**Comment by WoLpH** [comment]
The gods have granted us access, thank you very much!

As is the case with @RickDB my time is also limited (I severly underestimated the time restrictions involved with having a kid) and I'm actually not even using Hyperion anymore. I'll do what I can though, I'm going through some of the pull requests right now. Some are too involved for me to judge quickly, but a few of them I can fix :)

**Comment by RickDB** [comment]
Awesome 👍 

We probably need to look at getting Travis fixed as well, it currently errors out so hard to tell if a commit will break anything (or platform specific).

**Comment by RickDB** [comment]
Just an update as we have merged all the remaining pull requests and added @Paulchen-Panther to the team a while back which made some create contributions.
While we still need to push packaged releases like with Hyperion non-ng in the near future for easy installs think we are in a much better state then before.

Closing this one for now, not very active on the forum anymore so if there's anything can always open a new issue here or send PM.
